deno/payloads/v8/_interactions/modalSubmit.ts

```diff
@@ -0,0 +1,20 @@
+import type { APIBaseInteraction, InteractionType, APIModalActionRowComponent } from '../mod.ts';
+
+/**
+ * https://discord.com/developers/docs/interactions/message-components#component-object-component-structure
```

Comment:
make sure the link is correct

Reply:
Doesn't seem to be any specific docs for this so I'll remove it